### DISCUSSION TITLE
How to integrate with Nuxt3

### DISCUSSION THREAD

**Discussion by vanminhquangtri** [node-config/node-config#752] [General]
Hello,
When I migrate my app from nuxt 2 to Nuxt 3, it throws multi errors. I actually don't have any idea how to fix this. There's also no suggest on the internet, Nuxt docs page as well.
I hope to receive any advice.

This is my config folder. In nuxt 2 we don't need to create file default.json. But with Nuxt 3, without default.json, will have error Configuration property "env" is not defined.
![image](https://github.com/node-config/node-config/assets/70369658/8df2dfb7-16db-4d3f-ba48-71a3d04cfc7a)

This is my nuxt.config.ts
![image](https://github.com/node-config/node-config/assets/70369658/bfd42f79-46bf-480d-94c2-b92079baf029)

This is how I get the config inside component:
![image](https://github.com/node-config/node-config/assets/70369658/23406d45-4180-43c2-8e37-6c42734daa5d)

My start command is: cross-env NODE_ENV=production nuxt dev.
I set NODE_ENV=production but it alway return the value inside default.json, not value from production.json or dev.json or staging.json.

**Comment by jdmarshall** [comment]
I guess we don’t have many Nuxt users here. Did you ever solve this problem?

**Comment by jdmarshall** [comment]
One theory: How are you setting NODE_ENV? Within the application or from outside?
